WasteWar/Assets/Scripts/Camera Controls/Move.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
+
+public class Move : MonoBehaviour
+{
+    [SerializeField]
+    private Terrain terrain;
+    private Vector3 terrainSize;
```

Comment:
separate serialized fields with regular privates with space

Reply:
you mean with enter?